### ISSUE TITLE
Bug?: Can't change GC theme (Mojave)

### ISSUE THREAD

**Issue by bobsica** [GoldenCheetah/GoldenCheetah#3118]
I am able to change fonts in Athlete preferences, but I am unable to switch out of my current theme (I believe it is Mustang). I have GC unchecked in the Accessibility preferences in Mojave (checking it just slows down GC and the fix is to uncheck the box; with the box checked the theme will still not update). I am trying to change the theme by selecting one, hitting apply theme (nothing happens) and then saving. I'm not able to find a theme definition in any of the ini files. I'm using Mojave 10.14.5. Possibly a bug?

**Comment by amtriathlon** [comment]
In the Wiki is explained how it works: https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Preferences_Appearance
Themes are defined in the source, not in external files: https://github.com/GoldenCheetah/GoldenCheetah/blob/a86b0e9b74882ed83af3eb9293d97331ea95a500/src/Gui/Colors.cpp#L596
BTW, the idea is to use the tracker for bugs and features and the users forum for help.

**Comment by bobsica** [comment]
Yes, I had read the wiki and have tried to switch themes as described.  

Bug report: theme switching (as described in the link above) does not work in GC 3.5 1903 under Mojave 10.14.5. You can change fonts, you can NOT apply a change in theme.

**Comment by amtriathlon** [comment]
Which part of the 2 step process doesn't work according to the documentation?
1) Apply doesn't update the color list according to the selected theme
or
2) Save doesn't change the colors according to the color list

**Comment by bobsica** [comment]
#1. You can pick any theme, hit apply and nothing happens, that is the theme does not change. Even if you go on to save and then restart GC the theme doesn’t change. However, font changing is immediate and works fine.

**Comment by amtriathlon** [comment]
When you select a theme and click Apply the appearance doesn't change but the color list should be updated, you can check this in the Colors tab.
On save appearance should be updated and the new colors saved in GoldenCheetah.ini settings, [location is system dependent](https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ-DATA#where-are-settings-located).
Can you check which of these 2 updates is failing?

**Comment by bobsica** [comment]
1. Opened GC, tried to change theme, didn't work, quit
2. Opened GC, hit color tab, took noted colours, picked new theme, hit color tabs, noted colours had changed.
3. switched back to theme tab, hit apply theme, then save. Theme changed as anticipated.

I'm not sure what to tell you, but for my system, to "unstick" the theme appeared to take switching to the color tab and then back. After this I tried changing the theme several times and it worked, after repeated attempts at changing the theme without the switch to the color tab.

Given the fact that apparently there are no other reports of this issue I would say this is not worth pursuing unless others have the same issue. And if anyone reads this in the future and does try picking the color tab.

Thanks for your help resolving this. I'll leave the thread open in case you need any further information, but from my end you can close it.
